lib/domain/fixed_data.rb

```diff
@@ -0,0 +1,12 @@
+module Domain
+  class FixedData
+    attr_accessor :year,
+                  :rc_uplift
+  
+     def initialize(**fixed_data)
```

Comment:
You could use `ActiveModel::Model` as it provides this initialize behaviour: https://api.rubyonrails.org/classes/ActiveModel/Model.html